lib/omniauth-concur-oauth2/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module Concur
-    VERSION = "0.0.1"
+    VERSION = "0.0.2"
```

Comment:
This one is for you @wilkinsondesouza

Reply:
I appreciate that my rambling is being remembered 😄